requirements.txt

```diff
@@ -0,0 +1,7 @@
+requests == 2.10
+pexpect == 2.4
+urllib3 == 1.20
+pyOpenSSL == 0.14
+cryptography == 1.3.4
+idna == 2.0.0
+
```

Comment:
you can set the contents of this file to be `-e .` to use the versions in `setup.py`. also, you should not specify specific versions for a library. otherwise, the users of the library can't effectively use the library as well as other libraries due to version conflicts.